src/main/java/mate/hwdao/dao/impl/ManufactureDaoImpl.java

```diff
@@ -44,6 +43,10 @@ public Manufacturer update(Manufacturer manufacturer) {
 
     @Override
     public boolean delete(Long id) {
-        return Storage.listManufacturer.removeIf(x -> x.getId().equals(id));
+        return Storage.manufacturers
```

Comment:
why do we remove previous version?

Reply:
couse of bad pull request that i did.
 changed